modules/ocm-accesscontrol-tab.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// ocm/ocm-overview.adoc
+
+[id="ocm-accesscontrol-tab_{context}"]
+= Access control tab
+
+This tab allows the cluster owner to grant roles to other users.
+
+.Prerequisites
+
+. You must be the cluster owner or have the correct permissions to grant roles on the cluster.
+
+.Procedure
+
+. Select the **Grant role** button.
+. Enter the Red Hat account login for the user that you wish to grant a role on the cluster.
+. Select the **Grant role** button on the dialog box.
+. [I couldn't test past this point as no names worked to grant access.]
```

Comment:
L18 is this last step. If the username is correct, the dialog should close and will list the user has having "Cluster Editor" access.